src/components/interfaces/HMI_API.xml

```diff
@@ -634,6 +634,15 @@
   <element name="phoneNumber">
     <description> Optional phone number of intended location / establishment (if applicable) for SendLocation.</description>
   </element>
+  <element name="timeToDestination">
+      <description>Optional time to destination field for ShowConstantTBT</description>
+  </element>
+  <element name="turnText">
+      <description>Turn text for turnList parameter of UpdateTurnList</description>
+  </element>
+  <element name="navigationText">
+    <description>Navigation text for UpdateTurnList.</description>
+  </element>
```

Comment:
```suggestion
  <element name="turnText">
      <description>Turn text for turnList parameter of UpdateTurnList</description>
  </element>
```
It turns out that `navigationText` and `turnText` seems to reference the exact same field (`Turn.navigationText`), but it seems that `turnText` is the one used by Core, so `navigationText` should be removed since it is technically unused. Unfortunately, this will likely require a proposal revision.

Reply:
navigationText removed from the capabilities in  https://github.com/smartdevicelink/sdl_core/pull/3595/commits/bf17d2a0364aa2fb82b4318e839119fb7e28422d and from the API in https://github.com/smartdevicelink/sdl_core/pull/3595/commits/20b96576559f956c1c5eaec7110291e790cac28c